### PR TITLE
Add a reminder to notify maintainers about SPI changes

### PR DESCRIPTION
**Pull request by kokosing**
Add a reminder to notify maintainers about SPI changes

That way maintainers will have a chance to get to know early about SPI
changes, they will be able to respond to them and potentially notify all
other affected SPI consumers they know about.

### CODE REVIEW

**Review by electrum — CHANGES_REQUESTED** [review]
If we want this, we could add a bot that looks for revapi changes and posts them somewhere.

**Comment by electrum** [comment]
First commit looks good

**Comment by electrum** [comment]
We could have something automatically tag the PR anytime something changes the SPI POM.

File: core/trino-spi/pom.xml [hunk 1]
```diff
@@ -212,6 +212,7 @@
                                 </item>
                                 <!-- Backwards incompatible changes since the previous release -->
                                 <!-- Any exclusions below can be deleted after each release -->
+                                <!-- Remember to notify to all Trino maintainers before merging backward incompatible SPI changes -->
```
Comment (dain) [inline core/trino-spi/pom.xml:215]: This is not a policy we currently have.  Please bring this up with the maintainers group if you would like to add this policy.

**Comment by kokosing** [comment]
I am sorry, I have should mark this a draft. I posted this to facilitate the discussion. I had no intention to merge it fast.

**Comment by kokosing** [comment]
> We could have something automatically tag the PR anytime something changes the SPI POM.

Do you mean by posting a comment on github?

**Comment by electrum** [comment]
Sorry, I meant labels, not tags. For example, we have something that automatically tags PRs as `hive`.

**Comment by dain** [comment]
> I am sorry, I have should mark this a draft. I posted this to facilitate the discussion. I had no intention to merge it fast.

No problem.  Github is not a good place to have a discussion.  I suggest you post on slack group.

> > We could have something automatically tag the PR anytime something changes the SPI POM.
> 
> Do you mean by posting a comment on github?

I think he means add a label to the issue.

**Comment by kokosing** [comment]
I was under impression that adding tags does not trigger notifications.

**Pull request closed without merging**